### ISSUE TITLE
 `@class` is clobbered - this should be configurable

### ISSUE THREAD

**Issue by shogun70** [gustafnk/h-include#20]
If a `<h-include>` element already has `@class` then its value is **replaced** by "include_200" (or similar). 

I think it would be more appropriate to only replace (whitespace-separated) tokens within `@class` that match `/^include_\d+$/i`. 

Ideally there is a way to intercept success and failure handling to implement a completely different behavior, leaving the current behavior as the (preventable) default action. 

I also note (so I don't forget) that there are a few code paths where errors in transclusion won't be indicated in any way:
- basically any place in `show_content()` which returns or throws before the bottom of the function.
- fragment not found errors log a console warning but don't change the `<h-include>` class or content. 
  Some user-interaction might have triggered a `@src` change but the contents and appearance 
  still matches the previous `@src`. 
  I wonder if this should be considered a `404` error?
- recursion error (maybe a `412` error is appropriate?)

**Comment by gustafnk** [comment]
It's possible to override how @class is modified with the onEnd hook from https://github.com/gustafnk/h-include/pull/21

**Comment by gustafnk** [comment]
When https://github.com/gustafnk/h-include/issues/27 gets fixed, I think it would be valuable to document how to create your own configured custom element and maybe export a few custom hooks so that it's as easy as possible to configure [an element that inherits from] h-include.

**Comment by gustafnk** [comment]
@shogun70 What do you think of this? https://github.com/gustafnk/h-include/commit/a17667d5934a5d4890106b3baf3183ad6aad5a19

**Comment by gustafnk** [comment]
Fixed in https://github.com/gustafnk/h-include/pull/37
